integration/cri-o/cri-o.sh

```diff
@@ -74,6 +75,24 @@ do
 	sed -i '/'${testName}'/a skip \"'${skipCRIOTests[$testName]}'\"' "ctr_kata_integration_tests.bats"
 done
 
+# selectively skip tests depending on the version of cri-o we're testing with
+CRIO_VERSION=$(git status | head -n1 | awk '{print $NF}' | cut -d'-' -f 2)
+echo GOT CRIO VERSION $CRIO_VERSION
+if [ "$CRIO_VERSION" != "main" ]; then
+    if [ -z "$CRIO_VERSION" ]; then
+        echo "Unknown version of cri-o - skipping more tests"
+        CRIO_VERSION="1.21"
```

Comment:
I'd be more comfortable setting `CRIO_VERSION` to the latest release of the project.  In this case, 1.22.0 (see https://github.com/cri-o/cri-o/releases/tag/v1.22.0).

Reply:
Well, this is not the version we fall back to for running the test, this is the version we use to decide what test we enable or not.
1.21 is the first version we started testing with. And it is a version that we know have none of our fixes.

Using this number by default means we will skip all "unfixed" tests. If we're not sure what version of cri-o we're testing with, this is the safest. 

If we use "1.22" for instance, we will enable 3 more tests that are not fixed in 1.21, which means 3 failures if we're testing with a version of cri-o < 1.22

Reply:
Ack!  Thanks for the explanation.